systemvm/debian/opt/cloud/bin/cs/CsDhcp.py

```diff
@@ -69,7 +69,7 @@ def process(self):
 
         if not self.cl.is_redundant() or self.cl.is_master():
             if restart_dnsmasq:
-                CsHelper.service("dnsmasq", "restart")
+                CsHelper.service("dnsmasq", "reload")
```

Comment:
@andrijapanicsb 
"reload" works only if the service is started.
this is why we have line 74

Reply:
I'll need to test this all @weizhouapache , but afaik dnsmasq is started when VR is created?
Do you see a problem here? (save me some time from testing all this...)

Reply:
> I'll need to test this all @weizhouapache , but afaik dnsmasq is started when VR is created?
@andrijapanicsb it seems dnsmasq is disabled in routers. it will be started when dnsmasq.conf is generated.
https://github.com/apache/cloudstack/blob/master/systemvm/debian/opt/cloud/bin/setup/common.sh#L588-L595


> Do you see a problem here? (save me some time from testing all this...)